### PR TITLE
Fix db retention

### PR DESCRIPTION
**Pull request by RahulMahale**

### CODE REVIEW

**Comment by RahulMahale** [comment]
@vipulnsward _a please review.

File: lib/boxy-cap/tasks/database.rake [hunk 1]
```diff
@@ -24,8 +24,7 @@ namespace :db do
       within release_path do
         with rails_env: fetch(:rails_env) do
           execute :rake, 'db:backup'
-          execute :rake, 'db:cleanup'
-          no_of_days = fetch(:db_dump_retention).to_i
+	  execute :rake, 'db:cleanup', "NO_OF_DAYS=#{fetch(:db_dump_retention).to_i}"
```
Comment (vipulnsward) [inline lib/boxy-cap/tasks/database.rake]: space here

Comment (RahulMahale) [inline lib/boxy-cap/tasks/database.rake]: fixed. My vim is messed up with several plugins

**Comment by RahulMahale** [comment]
@vipulnsward can we merge this ?

**Comment by vipulnsward** [comment]
Yes , go ahead.

**Pull request merged**